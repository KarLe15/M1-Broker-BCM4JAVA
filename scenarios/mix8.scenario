## operation, component, scenario, uri
CREATE,subscriber,simple_1,subscriber_1
CREATE,subscriber,simple_2,subscriber_2
CREATE,subscriber,simple_2,subscriber_3
CREATE,subscriber,simple_3,subscriber_4
CREATE,subscriber,simple_3,subscriber_5
CREATE,publisher,no_waiting,publisher_1
CREATE,publisher,no_waiting,publisher_2
CREATE,publisher,intense,publisher_3
CREATE,publisher,intense_2,publisher_4
