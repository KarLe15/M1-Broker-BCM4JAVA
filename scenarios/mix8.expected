{
  "subscriber_1" : 3900,
  "subscriber_2" : 5200,
  "subscriber_3" : 5200,
  "subscriber_4" : 5200,
  "subscriber_5" : 5200
}
